### PR TITLE
Returns -> Return

### PR DESCRIPTION
**Pull request by fingolfin**
... in many docstrings and comments, to follow the general recommendation
for how docstrings should be phrases.

### CODE REVIEW

File: base/asyncevent.jl [hunk 1]
```diff
@@ -306,7 +306,7 @@ Waits until `testcb()` returns `true` or `timeout` seconds have passed, whicheve
 The test function is polled every `pollint` seconds. The minimum value for `pollint` is 0.001 seconds,
 that is, 1 millisecond.
 
-Returns :ok or :timed_out
+Return :ok or :timed_out
```
Comment (fredrikekre) [inline base/asyncevent.jl:309]:
```suggestion
Return `:ok` or `:timed_out`.
```

Comment (fredrikekre) [reply base/asyncevent.jl:309]: Perhaps also `Waits` -> `Wait` above while we are here.

**Review by fredrikekre — APPROVED** [review]

**Comment by fingolfin** [comment]
This patch was actually sitting since August 2021 on my HD. Not sure why I never submitted it. Had some conflicts which I mostly resolved by deleting them, so this patch will certainly miss some `Returns` cases, but just because there's more to be done doesn't meant this shouldn't be done ;-)

File: base/asyncevent.jl (hunk 1 above)
Comment (fingolfin) [reply base/asyncevent.jl]: Aye, there is plenty more to be done :-)

**Pull request merged**